js/src/components/google-mc-account-card/connected-google-mc-account-card.js

```diff
@@ -85,6 +97,44 @@ const ConnectedGoogleMCAccountCard = ( {
 		removeNotice( notice.id );
 	};
 
+	const disableNotifications = async () => {
+		const { notice } = await createNotice(
+			'info',
+			__(
+				'Disabling the new Product Sync feature, please wait…',
+				'google-listings-and-ads'
+			)
+		);
+
+		try {
+			await fetchDisableNotifications();
+			invalidateResolution( 'getGoogleMCAccount', [] );
+		} catch ( error ) {
+			createNotice(
+				'error',
+				__(
+					'Unable to disable new product sync. Please try again later.',
+					'google-listings-and-ads'
+				)
+			);
+		}
+
+		removeNotice( notice.id );
+	};
+
+	// Show the button if the status is "approved" and the Notification Service is not hidden.
+	const showDisconnectNotificationsButton =
+		! hideNotificationService &&
+		googleMCAccount.wpcom_rest_api_status === 'approved';
```

Comment:
Since this PR was merged before some of the changes were finished. We missed to update the constant here.

Reply:
Hey @puntope, thanks for reviewing. I don't quite get what should be updated here? This is the same as in the PR 2325 https://github.com/woocommerce/google-listings-and-ads/pull/2325/files#diff-b7a4b2fed5fcd9b3e8df480b7fa40000976fc614ddc77831ab62970ad9702a2fR126-R128. Maybe I missed something?

Reply:
I mean that instead of 'approved' we can use some constants like `GOOGLE_WPCOM_APP_CONNECTED_STATUS_APPROVED`

Reply:
Ah yeah, thanks for clarifying. Updated in 07ddeab.